src/llscan.cc

```diff
@@ -745,6 +740,12 @@ void LLScan::ScanMemoryRanges(FindJSObjectsVisitor& v) {
 #ifndef LLDB_SBMemoryRegionInfoList_h_
   MemoryRange* head = ranges_;
 
+  const uint64_t addr_size = process_.GetAddressByteSize();
+
+  // Pages are usually around 1mb, so this should more than enough
+  const uint64_t block_size = 1024 * 1024 * addr_size;
```

Comment:
block_size is only defined inside #ifndef LLDB_SBMemoryRegionInfoList_h_. If you have the new API then block_size is undefined and llscan.cc doesn't compile.
Ubuntu 16.10 now has an lldb-3.9 package with the new API.
(The path that uses the SBMemoryRegionInfo API also speeds things up by only scanning writable memory regions.)